### ISSUE TITLE
Uninstall Script does not work with 10.6

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#33]
```
I can not remove soundflower in 10.6, because the uninstall script crashes.
```

Original issue reported on code.google.com by `law.sky...@gmail.com` on 1 Sep 2009 at 5:45

**Comment by GoogleCodeExporter** [comment]
```
My current workaround is do download:
http://code.google.com/p/soundflower/source/browse/trunk/Tools/Uninstall%20Sound
flower.scpt
and run it manually.

```

Original comment by `law.sky...@gmail.com` on 1 Sep 2009 at 5:49

**Comment by GoogleCodeExporter** [comment]
```
OS 10.6 Support is available in SoundFlower 1.5.0, now available here:
http://code.google.com/p/soundflower/downloads/list
```

Original comment by `74obje...@gmail.com` on 1 Sep 2009 at 6:20
- Changed state: **Done**
